tests/acceptance/edit-test.js

```diff
@@ -0,0 +1,17 @@
+/* globals server */
+
+import { test } from 'qunit';
+import moduleForAcceptance from 'remember/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | edit');
+
+test('redirects to edit on click', function(assert) {
+  server.createList('reminder', 5);
+
+  visit('/reminders/1');
+  click('.edit-button');
+
```

Comment:
Can we make this test more verbose? By editing the text and checking that after you submit the edit that the text actually changed? We could also check that we transition the route.